### DISCUSSION TITLE
Shall we document every right click menu at every possible location?

### DISCUSSION THREAD

**Discussion by ReinerBforartists** [Bforartists/Bforartists#3482] [General]
Hey friends, 

Currently we have the right click menus at elements not documented in all cases. Instead we have a general hint in chapter 6.

RMB menus at tools.

This is in most cases enough information since the rmb menus at elements are pretty similar. But it has of course some downsides. The biggest plus is, the manual would become unbelievable bloated when we would add every possible right click menu to all the elements.

What are your thoughts? Document them in chapter 6. Or where you can call them? The latter would be a bigger operation ...

Kind regards
Reiner

**Comment by ghost** [comment]
Documentation is a already some work, so adding those would make it much harder, lets not do it imo

**Comment by ReinerBforartists** [comment]
Okay, then let's document such things in chapter 6.
